vite.config.ts

```diff
@@ -22,15 +27,35 @@ export default defineConfig({
 						'**/*.css',
 						'**/*.test.{js,jsx,ts,tsx}',
 						'**/__*.*',
-						// This is for server-side utilities you want to colocate next to
-						// your routes without making an additional directory.
-						// If you need a route that includes "server" or "client" in the
-						// filename, use the escape brackets like: my-route.[server].tsx
+						// This is for server-side utilities you want to colocate
+						// next to your routes without making an additional
+						// directory. If you need a route that includes "server" or
+						// "client" in the filename, use the escape brackets like:
+						// my-route.[server].tsx
```

Comment:
are you using a prettier config different than the one in this repo? this text should not have changed

Reply:
I have not touched the prettier config in this repo. When I run `npm run format`, there are about 10 different files that change, I'd guess from people who did not run format before merging changes. 
cc @kentcdodds 

This is what I see when I pull the epic-stack and run `npm run format` 

![image](https://github.com/epicweb-dev/epic-stack/assets/30846026/74c6969d-88b8-4d0c-9c98-1e1988078141)

Reply:
Yeah, that does happen sometimes. I'm also confused by prettier changing the comments here (I've never seen it do that), but I don't mind 🤷‍♂️

Reply:
I think that occurs coz people forget to install prettier on github ui (me the first). I don't know what's the best solution..an auto commit in a PR seems very annoying. an autocommit on merge seems unsafe. a pr check seems overkill